test/Microsoft.Azure.Devices.Edge.Test.Common/IotHub.cs

```diff
@@ -91,19 +91,20 @@ public async Task<Device> CreateEdgeDeviceIdentityAsync(string deviceId, Option<
                 Device parentDevice = await this.GetDeviceIdentityAsync(p, token);
                 string parentDeviceScope = parentDevice == null ? string.Empty : parentDevice.Scope;
                 Log.Verbose($"Parent scope: {parentDeviceScope}");
-                return new Device(deviceId)
-            {
-                Authentication = new AuthenticationMechanism()
-                {
-                    Type = authType,
-                    X509Thumbprint = x509Thumbprint
-                },
-                Capabilities = new DeviceCapabilities()
-                {
-                    IotEdge = true
-                },
-                ParentScopes = new[] { parentDeviceScope }
-            };
+                var result = new Device(deviceId)
+                                {
```

Comment:
Formatting seems odd.. was that VS?

Reply:
looks better aligned in VS (curly bracket is under 'D') but added a missing space to the next level